TokenSale/test/blockchainLabsNZ/burnable_tests.js

```diff
@@ -0,0 +1,118 @@
+let KNC = artifacts.require("KyberNetworkCrystal");
+
+const assertFail = require("./helpers/assertFail");
+const BigNumber = require("bignumber.js");
+
+let knc;
+
+let owner;
+
+// Taken from Migrations
+Date.prototype.getUnixTime = function() { return this.getTime()/1000|0 };
+if(!Date.now) Date.now = function() { return new Date(); }
+Date.time = function() { return Date.now().getUnixTime(); }
+
+
+contract("KNC", function(accounts) {
+  beforeEach(async () => {
+
+    owner = accounts[0];
+    var publicSaleStartTime = new Date('Sat, 16 Sep 2017 06:00:00 GMT').getUnixTime();
+    var publicSaleEndTime = new Date('Sun, 24 Sep 2017 06:00:00 GMT').getUnixTime();
+
+    knc = await KNC.new(
+        226000000000000000000000000, //Initial supply
```

Comment:
I'd suggest putting this number into a variable, you use it all over the place (19 times in this PR)